### PR TITLE
Update test-and-deploy-microsoft-365-apps.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about test deployment and what it means

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/9465

File: microsoft-365/admin/manage/test-and-deploy-microsoft-365-apps.md [hunk 1]
```diff
@@ -51,6 +51,10 @@ You can manage testing and deployment of purchased and licensed Microsoft 365 Ap
 
 8. If the app has a status of **Update pending**, you can click on the app to open the Manage pane and update the app.
 
+
+> [!NOTE]
+> Is this a test deployment is a administrative tag to determine if the app is stll in testing Phase. It has no technical impact.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/manage/test-and-deploy-microsoft-365-apps.md]:
```suggestion
> _Is this a test deployment_ is an administrative tag to determine if the app is still in testing phase. It has no technical impact.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@kwekuako Gentle reminder

**Pull request merged**